### ISSUE TITLE
Modpack updating "failed to commit instance"

### ISSUE THREAD

**Issue by Madis0** [PrismLauncher/PrismLauncher#262]
### Operating System

Windows

### Version of Prism Launcher

Prism Launcher 5.0

### Version of Qt

Qt 6.4.0

### Description of bug

Getting this error every time I try to update a modpack.
![image](https://user-images.githubusercontent.com/8611110/197393027-10b973ab-0d96-47b3-9d50-bbf2282a7e45.png)

### Steps to reproduce

1. Run Prism
2. Add Instance -> Modrinth -> Fabulously Optimized
3. Pick any older version
4. "OK"
5. Wait for it to download
6. Add Instance -> Modrinth -> Fabulously Optimized
7. Pick latest version
8. "Update existing instance"
9. "Yes"
10. Getting stuck at "removing old conflicting files..." and after about 5 seconds, the popup "Failed to commit instance, even after multiple retries. It is being blocked by something." appears
11. Clicking "OK", the  "removing old conflicting files..." popup still stays open and Abort is grayed out. Though it can be closed with the X.

### Suspected cause

- I copied settings and instances from PolyMC
- This happens for me always on Windows 11 but never on Manjaro Linux (installed via AUR)

Any logs I can give to help?

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by ghost** [comment]
this ^

**Comment by DioEgizio** [comment]
i can reproduce but only on windows

**Comment by Scrumplex** [comment]
Logs would be nice. In your data directory there should be `PrismLauncher-*.log` files. 0 being the lastest and 4 being the oldest. So try to reproduce the issue, close the launcher and send the -0.log

**Comment by Madis0** [comment]
[PrismLauncher-0.log](https://github.com/PrismLauncher/PrismLauncher/files/9847126/PrismLauncher-0.log)
edit: removed userid

```
   28.548 C "Failed to apply override from C:/Users/Madis/AppData/Roaming/PrismLauncher/instances/.LAUNCHER_TEMP/{5a1a4bd2-0e5d-4de7-984a-ea0376512b56} to C:/Users/Madis/AppData/Roaming/PrismLauncher/instances/Fabulously Optimized"
   28.549 C Reason: "File exists"
   28.549 W Failed to override "C:/Users/Madis/AppData/Roaming/PrismLauncher/instances/.LAUNCHER_TEMP/{5a1a4bd2-0e5d-4de7-984a-ea0376512b56}" to "C:/Users/Madis/AppData/Roaming/PrismLauncher/instances/Fabulously Optimized"
   28.549 D Failed to commit instance "Fabulously Optimized 4.4.3" Initiating backoff: 1
   29.047 C "Failed to apply override from C:/Users/Madis/AppData/Roaming/PrismLauncher/instances/.LAUNCHER_TEMP/{5a1a4bd2-0e5d-4de7-984a-ea0376512b56} to C:/Users/Madis/AppData/Roaming/PrismLauncher/instances/Fabulously Optimized"
   29.047 C Reason: "File exists"
```
Seems like improper handling of temp folders?

**Comment by RuiNtD** [comment]
Also seeing this issue on Windows 11 with a fresh install.

**Comment by Solverbolt** [comment]
[PrismLauncher-4.log](https://github.com/PrismLauncher/PrismLauncher/files/14769447/PrismLauncher-4.log)


This is still an issue as of 3-27-2024

Prism Launcher can no longer update an existing instance, you are forced to install a fresh one every time, and even that does not work at times.

**Comment by Trial97** [comment]
@Solverbolt update your prismlauncher: you are using 7.2 and this was fixed in 8.2(we can't port fixes to an old version)
